build/pipeline.yml

```diff
@@ -0,0 +1,54 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: microsoft/vscode-engineering
+      ref: main
+      endpoint: Monaco
+
+parameters:
+  - name: publishPackage
+    displayName: 🚀 Publish vscode-oniguruma
+    type: boolean
+    default: false
+
+extends:
+  template: azure-pipelines/npm-package/pipeline.yml@templates
+  parameters:
+    npmPackages:
+      - name: vscode-oniguruma
+
+        buildSteps:
+          - script: npm ci
+            displayName: Install dependencies
+
+          - script: npm run build-tsc
+            displayName: Compile
+
+          - script: npm run package
```

Comment:
Seems like this script already runs `npm run build-tsc` so you can remove the task on like 33.